### PR TITLE
Edit information website and navbar fix

### PR DESCRIPTION
**Pull request by Sizo**
I think this is enough for the demo. I have updated the nav bar, ensuring consistency across the tabs. All relevant data field have been added although I have iceboxed things like error handling...

### CODE REVIEW

File: Source/Web/app/information.js [hunk 1]
```diff
@@ -425,7 +482,7 @@ function dispOrch(id) {
           "" +
```
Comment (Letanyan) [inline Source/Web/app/information.js]: Assigned Workers don't seem to show up. I was testing with Apple Farm - Block H Orchard.

File: Source/Web/app/information.js [hunk 2]
```diff
@@ -616,6 +686,12 @@ function dispWork(id) {
         "<div class='form-group'><label class='control-label col-sm-2' for='text'>Worker Surname:</label>" +
         "<div class='col-sm-9'><input type='text' class='form-control' id='workSName'></div> </div>" +
         "" +
+	"<div class='form-group'><label class='control-label col-sm-2' for='text'>Identity Number:</label>" +
+	"<div class='col-sm-9'><input type='text' class='form-control' id='workID'></div> </div>" +
+        "" +
+	"<div class='form-group'><label class='control-label col-sm-2' for='text'>Phone Number:</label>" +
+	"<div class='col-sm-9'><input type='text' class='form-control' id='workContactNo'></div> </div>" +
+        "" +
         "<div class='form-group'><label class='control-label col-sm-2' for='sel1'>Assigned Orchard:</label>" +
```
Comment (Letanyan) [inline Source/Web/app/information.js]: Assigned Orchards don't show up. For example Bruce Wayne is assigned to 3 orchards but they don't show up here.

File: Source/Web/app/information.js [hunk 3]
```diff
@@ -653,7 +729,13 @@ function dispWork(id) {
...
+		    "" +
+                    "<div class='form-group'><label class='control-label col-sm-2' for='text'>Identity Number:</label>" +
+                    "<div class='col-sm-9'><p class='form-control-static'>" + snapshot.val().idNumber + "</p> </div> </div>" +
+		    "" +
+		    "<div class='form-group'><label class='control-label col-sm-2' for='text'>Phone Number:</label>" +
+                    "<div class='col-sm-9'><p class='form-control-static'>" + snapshot.val().phoneNumber + "</p> </div> </div>" +
+		    "" +
                     "<div class='form-group'><label class='control-label col-sm-2' for='sel1'>Assigned Orchard:</label>" +
                     "<div class='col-sm-9'><span id='workOrchDisp'></span></div></div>" +
                     "" +
```
Comment (Letanyan) [inline Source/Web/app/information.js]: When the foreman option is selected it must ask for the phone number and not the email anymore.

File: Source/Web/app/information.js (hunk 2 above)
Comment (Sizo) [reply Source/Web/app/information.js]: Ow the working for collection isn't getting an update. Will fix that now

**Pull request merged**